queue_services/entity-filer/src/entity_filer/filing_processors/conversion.py

```diff
@@ -56,35 +60,31 @@ def process(
     filing_meta.conversion = {}
     if not (conversion_filing := filing.get("filing", {}).get("conversion")):
         raise BusinessException(
-            f"CONVL legal_filing:conversion missing from {filing_rec.id}"
-        )
+            error=f"CONVL legal_filing:conversion missing from {filing_rec.id}",
+            status_code=HTTPStatus.BAD_REQUEST)
```

Comment:
`BusinessException` is missing the required parameter `status_code` in a few places in this file. I have added the HTTPStatus code that I think makes the most sense in each case.